charts/stable/cloudreve/values.yaml

```diff
@@ -1,32 +1,58 @@
 image:
   pullPolicy: IfNotPresent
-  repository: xavierniu/cloudreve
-  tag: latest@sha256:cdd00367bbdf3247e15feebfbde354237586a308c6fac730856670dd31b44b47
+  repository: cloudreve/cloudreve
+  tag: 3.8.3@sha256:a9e2373b7ca59bc43b9a4ceb50e7f6b80539c1732a9439b96dfe1aec6354a6c5
 persistence:
   avatarpath:
     enabled: true
     mountPath: /cloudreve/avatar
   config:
     enabled: true
-    mountPath: /cloudreve/config
-  databasepath:
-    enabled: true
-    mountPath: /cloudreve/db
+    targetSelector:
+      main:
+        main:
+          mountPath: /cloudreve
+        init:
+          mountPath: /conf-init
   uploadpath:
     enabled: true
     mountPath: /cloudreve/uploads
-portal:
-  open:
-    enabled: true
 securityContext:
   container:
     readOnlyRootFilesystem: false
     runAsGroup: 0
     runAsUser: 0
+workload:
+  main:
+    podSpec:
+      initContainers:
+        init:
+          enabled: true
+          # Only needs to run on upgrade and install..
+          type: init
+          imageSelector: image
+          command:
+            - "/bin/sh"
+            - "-c"
+            - |
+              set -x
+              ls -lasth /cloudreve
+              cp -a /cloudreve/cloudreve /conf-init/cloudreve
+              ls -lasht /conf-init
+      containers:
+        main:
+          probes:
+            liveness:
+              path: "/"
+            readiness:
+              path: "/"
+            startup:
+              path: "/"
 service:
   main:
     ports:
       main:
         port: 5212
```

Comment:
If there isn't a way to set the internal port of the container (eg with an env) you need the targetPort here.
as the port can be changed by the user at any time. (targetPort defaults to port if empty)

Reply:
It looks like the port can be set as an [option in conf.ini](https://github.com/cloudreve/Cloudreve/blob/88409cc1f0a0a55b7ea8f0aee15e82876260b2d8/pkg/conf/conf.go#L77), but I don't see a way to leverage that without taking ownership of the whole file in a configmap. If you agree I'll just add the targetPort.